dts/bindings/rtc/atmel,sam0-rtc.yaml

```diff
@@ -22,10 +22,4 @@ properties:
       description: mmio register space
       generation: define
       category: required
-
-    clock-generator:
```

Comment:
Please keep this

Reply:
Ok, I will put it back in, but right now (pending something like #15585) actually setting it would break the driver: it currently hard codes the clock frequency as the one from GCLK 0.